app/models/grant.rb

```diff
@@ -0,0 +1,5 @@
+class Grant < ActiveRecord::Base
+  has_many :items, dependent: :destroy
```

Comment:
Please remove the dependent destroy. We will probably disallow destruction of grants.